elastic/kibana
# [RAM] Stack alerts page fixes and improvements

## Summary

- Renames `GlobalAlertsPage` to `StackAlertsPage` to better account for the change in scope
- Renames the `triggerActionsUi.globalAlertsPage` flag to `triggerActionsUi.stackAlertsPage`
- Toggles the flag on by default
- Updates the tech preview badge to match the other badges used across triggers_actions_ui
- Fixes the feature cell rendering logic not displaying Observability alerts correctly
- Other small fixes and improvements

## To verify

1. Navigate to the `Stack management > Alerts` page (the page should now be visible without a the feature flag turned on manually)
2. Check that the tech preview badge appears correctly and shows the tooltip on hover
3. Check that Observability alerts show an `Observability` badge in the `Feature` column
